it/uk/gov/hmrc/thirdpartyapplication/repository/ApplicationRepositoryISpec.scala

```diff
@@ -3006,7 +3006,7 @@ class ApplicationRepositoryISpec
       val app = anApplicationData(applicationId).copy(access = access)
       await(applicationRepository.save(app))
 
-      val event = ProductionAppPrivacyPolicyLocationChanged(UpdateApplicationEvent.Id.random, applicationId, LocalDateTime.now, gkUser, oldLocation, newLocation, adminEmail)
+      val event = ProductionAppPrivacyPolicyLocationChanged(UpdateApplicationEvent.Id.random, applicationId, LocalDateTime.now, gkUser, oldLocation, newLocation)
```

Comment:
I think this should use a collaborator actor (and several down below too)

Reply:
Fixed